azure-toolkit-libs/pom.xml

```diff
@@ -54,62 +54,67 @@
+        <azure.aad.version>1.6.7</azure.aad.version>
         <azure.applicationinsights.version>2.6.3</azure.applicationinsights.version>
         <azure.applicationinsights-web.version>[2.0,)</azure.applicationinsights-web.version>
         <azure.azure-mgmt-appplatform.version>1.0.0-beta</azure.azure-mgmt-appplatform.version>
         <azure.client.version>1.7.12</azure.client.version>
-        <azure.identity.version>1.3.0-beta.2</azure.identity.version>
-        <azure.resourcemanager.version>2.4.0</azure.resourcemanager.version>
-        <azure.storage.version>8.0.0</azure.storage.version>
-        <azure.version>1.38.1</azure.version>
-        <azure.core.http.okhttp.version>1.6.1</azure.core.http.okhttp.version>
-        <azure.storage-file-share.version>12.9.0</azure.storage-file-share.version>
+        <azure.identity.version>1.3.0</azure.identity.version>
+        <azure.resourcemanager.version>2.5.0</azure.resourcemanager.version>
+        <azure.storage.version>8.6.6</azure.storage.version>
+        <azure.version>1.41.1</azure.version>
+        <azure.eventhubs.version>3.2.2</azure.eventhubs.version>
+        <azure.eventhubs-eph.version>3.2.2</azure.eventhubs-eph.version>
+        <azure.core.http.okhttp.version>1.6.2</azure.core.http.okhttp.version>
+        <azure.storage-file-share.version>12.10.0-beta.1</azure.storage-file-share.version>
 
-        <aspectj.version>1.8.13</aspectj.version>
+        <aspectj.version>1.9.7.M3</aspectj.version>
         <codehaus.plexus-utils.version>3.3.0</codehaus.plexus-utils.version>
         <commons.codec.version>1.15</commons.codec.version>
         <commons.collections4.version>4.4</commons.collections4.version>
         <commons.exec.version>1.3</commons.exec.version>
-        <commons.io.version>2.8.0</commons.io.version>
-        <commons.lang3.version>3.11</commons.lang3.version>
-        <commons.net.version>3.6</commons.net.version>
+        <commons.io.version>2.9.0</commons.io.version>
+        <commons.lang3.version>3.12.0</commons.lang3.version>
+        <commons.net.version>3.8.0</commons.net.version>
         <dom4j.version>2.1.3</dom4j.version>
-        <gson.version>2.8.6</gson.version>
-        <guava.version>30.1-jre</guava.version>
-        <httpclient.version>4.5.9</httpclient.version>
-        <jackson.version>2.12.1</jackson.version>
+        <error.prone.core.version>2.7.1</error.prone.core.version>
+        <gson.version>2.8.7</gson.version>
+        <guava.version>30.1.1-jre</guava.version>
+        <httpclient.version>4.5.13</httpclient.version>
+        <jackson.version>2.12.3</jackson.version>
+        <jacoco.version>0.8.7</jacoco.version>
         <jacoco.version>0.8.5</jacoco.version>
-        <jansi.version>1.17.1</jansi.version>
-        <jetbrains.annotations.version>19.0.0</jetbrains.annotations.version>
+        <jansi.version>2.3.2</jansi.version>
+        <jetbrains.annotations.version>21.0.1</jetbrains.annotations.version>
```

Comment:
Why we have `jetbrains.annotations` dependency in lib?

Reply:
common-lib used it. I guess it should be used for Intellij.
we can sync who add this in it and try to push him remove it.

Reply:
We will remove such dependency later.

Reply:
good to me.